### PR TITLE
Update travis and remove legacy template

### PR DESCRIPTION
**Pull request by ongspxm**
as proposed, this is the second part of the frontend upstream/master merge
- this one covers the travis changes 
- java side changes to remove the copying of the `repo_report` old template.

### CODE REVIEW

**Comment by eugenepeh** [comment]
@ongspxm merged #153, please refer to #163 for mergeability.

**Comment by ongspxm** [comment]
imma redo this thing, dun merge first, this is a little messy

**Comment by ongspxm** [comment]
@eugene ready to merge, i redid everything already

File: template/index.html [hunk 1]
```diff
@@ -1,2 +1 @@
...
-
```
Comment (eugenepeh) [inline template/index.html]: this change doesn't seems necessary to me

File: template/static/css/style.css [hunk 2]
```diff
@@ -1,3 +1,2 @@
...
-
```
Comment (eugenepeh) [inline template/static/css/style.css]: same as this

File: template/static/css/_colors.css [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (eugenepeh) [inline template/static/css/_colors.css]: same as this

File: template/static/js/v_summary.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ function comparator(fn) {
     const b1 = fn(b);
     if (a1 === b1) {
       return 0;
-    } else if (a1 < b1) {
+    } if (a1 < b1) {
```
Comment (eugenepeh) [inline template/static/js/v_summary.js]: what's the rationale of this change?

**Comment by eugenepeh** [comment]
btw, update the PR title.

Also regarding the output, is it awaiting zip function to display the results on netlify?

File: template/index.html (hunk 1 above)
Comment (ongspxm) [reply template/index.html]: this is generated file, previously i used to manually add a newline to the file to pass the `config/travis/run-checks.sh`. the `no-line-end` part.

now i changed the "compiler" to print out the line-end chracter, so dun need the newline temporary hack anymore.

File: template/static/css/style.css (hunk 2 above)
Comment (ongspxm) [reply template/static/css/style.css]: same for this.

File: template/static/css/_colors.css (hunk 3 above)
Comment (ongspxm) [reply template/static/css/_colors.css]: i think i didn't do a clean build, i will redo this.

File: template/static/js/v_summary.js (hunk 4 above)
Comment (ongspxm) [reply template/static/js/v_summary.js]: oh, this change was made in alignment of the airbnb javascript style guide. https://github.com/airbnb/javascript#blocks--no-else-return

**Comment by ongspxm** [comment]
> Also regarding the output, is it awaiting zip function to display the results on netlify?

no need, running it on netlify is like running it on a server, can just run straight away.

**Comment by eugenepeh** [comment]
just another highlight, it seems that the top of the code panel vertical scroll bar is not visible
